controllers/internal/crutils/static_resource_generator.go

```diff
@@ -397,6 +404,76 @@ func (c *IBMBlockCSI) GenerateCSIAddonsReplicatorClusterRoleBinding() *rbacv1.Cl
 	}
 }
 
+func (c *IBMBlockCSI) GenerateVolumeGroupClusterRole() *rbacv1.ClusterRole {
+	return &rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: config.GetNameForResource(config.CSIVolumeGroupClusterRole, c.Name),
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{volumeGroupApiGroup},
+				Resources: []string{volumeGroupsResources},
+				Verbs:     []string{verbGet, verbList, verbWatch, verbCreate, verbUpdate, verbPatch, verbDelete},
+			},
+			{
+				APIGroups: []string{volumeGroupApiGroup},
+				Resources: []string{volumeGroupsStatusResource},
+				Verbs:     []string{verbGet, verbUpdate, verbPatch},
+			},
+			{
+				APIGroups: []string{volumeGroupApiGroup},
+				Resources: []string{volumeGroupsFinalizersResource},
+				Verbs:     []string{verbUpdate},
+			},
+			{
+				APIGroups: []string{volumeGroupApiGroup},
+				Resources: []string{volumeGroupClassesResource},
+				Verbs:     []string{verbGet, verbList, verbWatch},
+			},
+			{
+				APIGroups: []string{volumeGroupApiGroup},
+				Resources: []string{volumeGroupContentsResource},
+				Verbs:     []string{verbGet, verbList, verbWatch},
```

Comment:
can you add to `volumeGroupContentsResource` the `Create` verb? (I know it's not written in the sidecar I will add it this week)?